samples/SampleStartups/StartupHelloWorld.cs

```diff
@@ -6,26 +6,58 @@
 
 namespace SampleStartups
 {
-    public class StartupHelloWorld : StartupBase
+    public class Program
     {
-        // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
-        public override void Configure(IApplicationBuilder app)
+        public static void Main()
         {
-            app.Run(async (context) =>
-            {
-                await context.Response.WriteAsync("Hello World!");
-            });
+            var host = new WebHostBuilder()
+                .Run(async context =>
+                {
+                    await context.Response.WriteAsync("Hello World");
+                });
+
+            host.WaitForShutdown();
+        }
+
+        public static void MainWithPort()
+        {
+            var host = new WebHostBuilder()
+                .Run(8080, async context =>
```

Comment:
> We've avoided specifying just a port because of the ambiguity over what the IP/Host is. Localhost? */Any? Loopack? IPv6Loopback? There's never been agreement for what the default host/ip should be.
  
I know, I was there, I'm forcing the discussion by picking `Any`.

> I also don't see this extension being used outside of demoware, no real app is this simple. I'd rather have the RunApplication variant as Run.

Agreed. Or very simple applications.